WeakAurasOptions/AceGUI-Widgets/AceGUIWidget-WeakAurasDisplayButton.lua

```diff
@@ -885,6 +893,11 @@ local methods = {
       notCheckable = true,
       func = function() OptionsPrivate.ExportToTable(self.data.id) end
     });
+     tinsert(self.menu, {
+      text = L["Ignore Wago Update"],
+      checked = function() return OptionsPrivate.IsWagoUpdateIgnored(self.data.id) end,
+      func = function() self.callbacks.IgnoreWagoUpdate(self.data.id) end
+    });
```

Comment:
This shouldn't be shown if there is no wago url for this aura

I'm not sure if this option belong to the contextual menu, if it was in the Information tab you would see wago url this aura is linked to, i think it make more sense to move it there

Reply:
my first thought was having it in the information tab aswell but since i remembered it being in the dropdown before i thought best keep it there. also the options in the information tab didn't seem fitting since there is only compatibility and debug options so it'd prob need a new tab?